source/unified-test-format/tests/valid-pass/poc-queryable-encryption.yml

```diff
@@ -0,0 +1,84 @@
+description: poc-queryable-encryption
+
+schemaVersion: "1.23"
+
+runOnRequirements:
+  - minServerVersion: "7.0"
+     csfle: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+      autoEncryptOpts:
+        kmsProviders:
+          local:
+            key: Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name poc-queryable-encryption
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+  - database:
+      id: &database1 database1
+      client: *client0
+      databaseName: &database1Name keyvault
+  - collection:
+      id: &collection1 collection1
+      database: *database0
```

Comment:
Shouldn't this be `database1`? Consider using more descriptive names for the entities to make this more self-documenting, as I can imagine this would be an easy mistake to make in future test files that use these entity definitions as a base.